### ISSUE TITLE
Icon request - Update Wolfram Mathematica .wl mimetype

### ISSUE THREAD

**Issue by jose-a-sa** [numixproject/numix-icon-theme#1303]
Build on the Numix .nb mimetype. 

| Application name | Icon name | Desktop file name | Original icon |
| --- | :-- | :-- | :-- |
| `.wl mimetype`| `application-vnd.wolfram.wl.svg` | `n/a` | ![vnd wolfram wl-64](https://user-images.githubusercontent.com/13206784/39371308-3c5e16f0-4a39-11e8-9102-e5eb78321791.png) |
| `.cdf mimetype`| `application-vnd.wolfram.cdf.svg` | `n/a` | ![vnd wolfram cdf-64](https://user-images.githubusercontent.com/13206784/39371453-a82c6864-4a39-11e8-91ca-14b51bc7a072.png) |

Description: `Wolfram Mathematica`

Website: `https://www.wolfram.com/language/`

**Comment by jose-a-sa** [comment]
That will do but a new icon for .wl files (script based) would be nice.
Thanks

**Comment by Foggalong** [comment]
@palob Yeah, the original issue doesn't do a great job explaining but Wolfram Language and CDF files aren't the same as Mathematica, though the three all do interact with each other.
